auth_mellon_config.c

```diff
@@ -1099,6 +1099,14 @@ const command_rec auth_mellon_commands[] = {
         " '/'."
         ),
     AP_INIT_TAKE1(
+        "MellonCookieSameSite",
+        ap_set_string_slot,
```

Comment:
Yes, that makes sense as you wouldn't want to give a user a false sense of security which was invalidated by a typo.